regulations/templates/regulations/preamble-chrome.html

```diff
@@ -10,11 +10,19 @@
 
 {% block wayfinding %}
 <span class="subpart">{{doc_number}}</span>
-<span id="active-title">Active Title</span>
+<span id="active-title"></span>
 {% endblock %}
 
 {% block header-secondary %}
-<span class="effective-date"><strong>Published Date:</strong> A Date</span>
+<div class="preamble-header">
+  <span>Actions:</span>
+  <div class="read-proposal active-mode">
+    Read the proposal
+  </div>
+  <div class="write-comment">
+    Write a comment
+  </div>
```

Comment:
Heads up -- in a later revision, we'll probably want these to be `a`s or similar for accessibility.